test/adapters/qunit2.js

```diff
@@ -51,18 +53,19 @@ QUnit.test("it works", function(assert) {
 		},
 		equiv: function() {
 			stats.equiv += 1;
-		}
+		},
+		config : { current : { assert : fakeAssert }}
 	};
-
+	
 	var adapted = adapter(fakeQunit);
 
 	fakeQunit.test("test", function() {});
 	adapted.pauseTest();
 	adapted.resumeTest();
-	adapted.assertOK();
+	adapted.assertOK(true);
 	adapted.equiv();
 
-	assert.deepEqual(stats, {
+	QUnit.assert.deepEqual(stats, {
```

Comment:
Was this changed for a reason? Seems like it should reference the `assert` that was passed in.

Reply:
You are correct. I have reverted the assertOk(true) to just assertOk().

Under the hood this should call getCurrentAssert().ok(assertion, message); in the adapter, which should grab the assert that we pushed onto the stack